src/ophys_etl/transforms/data_loaders.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import numpy as np
+from collections import namedtuple
+
+motion_border = namedtuple('motion_border', 'left right up down')
+
+
+def get_max_correction_values(x_series: pd.Series, y_series: pd.Series,
+                              max_shift: float = 30.0) -> motion_border:
+    """
+    Gets the max correction values in the cardinal directions from a series
+    of correction values in the x and y directions
+    Parameters
+    ----------
+    x_series: pd.Series:
+        A series of movements in the x direction
+    y_series: pd.Series:
+        A series of movements in the y direction
+    max_shift: float
+        Maximum shift to allow when considering motion correction. Any
+        larger shifts are considered outliers.
+
+    Returns
+    -------
+    motion_border
+        A named tuple containing the maximum correction values found during
+        motion correction workflow step. Saved with the direction is the order
+        [Left, Right, Up, Down] and also with names.
+
+    """
+    # validate if input columns match supported versions of motion correction
+    # files
+    if max_shift <= 0:
```

Comment:
Nit: We actually are looking at the range from [-max_shift, max_shift], so we could just apply `abs` to all the inputs and not raise this error.

Reply:
Added absolute value to this and an inline comment noting the change.